addons/website_form/static/src/snippets/s_website_form/000.js

```diff
@@ -65,7 +65,12 @@ odoo.define('website_form.s_website_form', function (require) {
             // Because, using t-att- inside form make it non-editable
             var $values = $('[data-for=' + this.$target.attr('id') + ']');
             if ($values.length) {
-                var values = JSON.parse($values.data('values').replace('False', '""').replace('None', '""').replace(/'/g, '"'));
+                var values = JSON.parse($values.data('values')
```

Comment:
Since you touch the line, you could take the chance to convert it.
```suggestion
                const values = JSON.parse($values.data('values')
```
(Sometimes we also take the chance to remove jQuery usage to help later when we will remove it but it's not necessary here, the `var` -> ` const` however is quite simple and no risk as you touched the line already)